### ISSUE TITLE
Regex over semantics?

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#54]
https://web.archive.org/web/20191224134416/https://www.clips.uantwerpen.be/pages/pattern-search

**Comment by Decagon** [comment]
The pattern search that you linked to would make things much much easier. We should definitely consider incorporating this into the application.

**Comment by j10sanders** [comment]
agreed with @Decagon!  I'm going to start messing around with it.  

I wonder which issues it would be most helpful for?
